web/client/reducers/config.js

```diff
@@ -47,6 +47,34 @@ function mapConfig(state = null, action) {
             return assign({}, state, {
                 mousePointer: action.pointer
             });
+        case LAYER_LOADING: {
+            let loadingLayers = assign((state && state.loadingLayers) || {});
+            loadingLayers[action.layerId] = true;
```

Comment:
I think you are mutating the original state.loadingLayers object here, you should avoid that: assign({}, (state && state.loadingLayers) || {}) should do the trick